input/pagecontent/reference.md

```diff
@@ -190,7 +194,7 @@ The following restrictions apply to rules:
 * All rules in FSH begin with an asterisk (`*`) symbol followed by at least one space.
 * All rules must begin on a new line.
 * Rules cannot be preceded by non-whitespace characters on a line.
-* Whitespace characters prior to the initial asterisk (`*`) are meaningful. See [Indented Rules](#indented-rules) below.
+* Whitespace characters prior to the initial asterisk (`*`) are meaningful. Rules must left-justified unless using [indented rule syntax](#indented-rules) {%include tu.html%}.
```

Comment:
My only concern here is that it kind of seems like it is saying the requirement that rules be left-justified (except indenting) is TU.  But I think we need to make the requirement for left-justification _normative_ since it fails otherwise (whether or not you use indenting) -- and because it would be a breaking change if we tried to make it normative _later_.

Do you think we need to clarify this somehow or am I being too sensitive to potential misunderstandings?

Reply:
If we make left-justification normative, it means all rules must be left justified, period. Since we want to allow indentation, then we can't do that. So I think it is correct as written. The fact that SUSHI fails on indented rules just means that SUSHI is aligned to TU features.

Reply:
> If we make left-justification normative, it means all rules must be left justified, period.

Yeah, that's not what I meant, but I now realize it is what I _said_.  And I can't think of a way to say it differently.  So... OK.

Reply:
Comments on normative versus trial use:
- Parameterized RuleSets -- If you are uncomfortable with this, we can do TU
- Soft Indexing -- I agree, it can go normative
- Quantity syntax allowing non-UCUM code system units --  is there any risk if we go normative?
- Quantity syntax allowing human-readable units --   is there any risk if we go normative?
- Support for integer64 -- no risk, so I prefer normative
- Caret paths for CodeSystems -- I'm ok with calling this TU 
- Support for prerelease versions of R5 --  CodeableReference is TU. The fact that SUSHI supports R5 is not otherwise relevant for the language spec.

Reply:
* Parameterized RuleSets -- let's go `TU`, as I'd like to see if we might improve on what we have.
* Quantity syntax (both changes) -- they seem low risk, so I'm good w/ normative.
* Support for integer64 -- I think we might want to make it TU because it is in R5 and R5 is not yet released.  What happens if integer64 gets rejected or renamed in the R5 ballot?
* Support for prerelease versions of R5 -- I hear what you're saying, but I was responding to [3.2.3 Dependency on FHIR version](https://build.fhir.org/ig/HL7/fhir-shorthand/reference.html#dependency-on-fhir-version), which says:
    > FSH supports pre-release FHIR R5, but support for pre-release versions is still experimental.
    
    If we keep that wording, then it sounds like `TU` to me (i.e., "experimental"). The other alternative is to strike it or reword it. I agree that in many respects, FHIR version (post 4.0) is primarily an implementation detail.

Reply:
OK. Implemented those changes.